source/process/community-process.rst

```diff
@@ -0,0 +1,101 @@
+Community Monitoring Process
+============================
+
+This document outlines the process for monitoring and responding to community questions in various channels.
+
+Community Product Manager
+--------------------------
+
+Every week, we rotate which PM is responsible for making sure community questions get answered. We rotate alphabetically (for example, Eric -> Jason -> Lindsay).
+
+When responding to issues:
+
+1. Identify what type of post it is:
+
+  - Troubleshooting
+  - Feature Request
+  - Bug
+  - Developer/Product Question
+
+2. Ask questions to narrow down or understand the issue better
+
+3. Respond:
+
+  - Look for related documentation, Jira ticket, feature request, etc.
+  - If there is no related documentation:
+
+    - For reproducible bugs, create a Jira ticket
+    - For product questions or troubleshooting, update docs.mattermost.com if appropriate
```

Comment:
Should we add a note in here about approaching an appropriate dev if the question is technical?